tools/rpi-sign-bootcode

```diff
@@ -196,34 +230,50 @@ def main():
         2711 mode:
         rpi-sign-bootcode --debug -c 2711 -i bootcode.bin.clr -o bootcode.bin -k 2711_rsa_priv_0.pem -n 0 -m bootcode-production.key
 
-        2712 C1 and D0 mode:
-          * HMAC not included on 2712
+        2712 A0, B0 mode:
+        rpi-sign-bootcode --debug -c 2712 -i bootcode.bin.clr -o bootcode.bin -k 2712_rsa_priv_0.pem -n 0 -m bootcode-production.key
+
+        2712 B1 mode:
```

Comment:
Old text said "2712 C1 and D0 mode:" so should this be changed to "2712 B1 (and newer) mode:" ?

Reply:
Doesn't  matter